MailFunctions.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+require_once 'Core/Dotenv.php';
```

Comment:
Let's create this as a service so this can be reused later.
Create a class and necessary methods , then we will pass necessary information via arguments.